app/uk/gov/hmrc/cataloguefrontend/DependencyExplorerController.scala

```diff
@@ -84,13 +84,13 @@ class DependencyExplorerController @Inject() (
     }
 
   def search(
-    team: String = "",
-    flag: String,
-    `scope[]`: Seq[String],
-    `repoType[]`: Seq[String],
     group: String,
     artefact: String,
-    versionRange: String,
+    team: Option[String],
+    flag: Option[String],
+    `scope[]`: Option[List[String]],
+    `repoType[]`: Option[List[String]],
+    versionRange: Option[String],
```

Comment:
Can `versionRange` come after `group` and `artefact`, since they're a triple (the `gav`) and all identify the dependency match (the other filters are all matching the usage of the dependencies)